_data-prepper/pipelines/configuration/processors/otel-trace-raw.md

```diff
@@ -10,23 +10,34 @@ nav_order: 45
 
 ## Overview
 
-This processor is a Data Prepper event record type replacement of `otel_trace_raw_prepper` (no longer supported since Data Prepper 2.0). The processor fills in trace group related fields including the following.
+This processor completes trace-group-related fields in all incoming Data Prepper span records by state caching the root span information for each `tradeId`. This processor includes the following parameters:
```

Comment:
Can you please include processor name here?
The `otel_trace_raw` processor completes ...